spec/system/subject_spec.rb

```diff
@@ -86,44 +84,34 @@
     expect(page).to have_text('T1')
   end
 
-  context "when logged in" do
-    before do
-      sign_in create(:user)
-    end
-
-    it "can review subjects" do
-      visit subject_path(gal1)
+  it "can review subjects" do
+    user = create(:user)
 
-      expect(page).to have_text('Sin calificar')
-      click_button('star_outline', match: :first)
+    visit subject_path(gal1)
 
-      expect(page).to have_text('Puntuación: 5.0')
+    assert_text 'Sin calificar'
```

Comment:
```suggestion
    expect(page).to have_text('Sin calificar')
```

para ser consistentes?

Reply:
Arreglado en [9f00455](https://github.com/cedarcode/mi_carrera/pull/757/commits/9f004558707f90263d1a5e9e3ef7a612b2225eba)